docs/meshstack.aws.index.md

```diff
@@ -687,4 +720,4 @@ def main(event, context):
 
 This Lambda will be invoked during account provisioning and downgrades the `MeshstackAccountAccessRole`.
 
-> Depending of your mode of operation (usage of external Account Vending Machine) these "minimal rights" can be adapted and further restricted. Please [contact us](https://support.meshcloud.io) for more details on reducing these rights.
+> Depending of your mode of operation (usage of external Account Vending Machine or configuration of AWS Control Tower enrollment) these "minimal rights" can be adapted and further restricted. Please [contact us](https://support.meshcloud.io) for more details on reducing these rights.
```

Comment:
```suggestion
> Depending of your mode of operation (usage of external Account Vending Machine or configuration of AWS Control Tower enrollment) these "minimal rights" can be adapted and further restricted. Please [contact us](mailto:support@meshcloud.io) for more details on reducing these rights.
```
Not even sure if a mailto links works here, please test it. But `support.meshcloud.io` doesn't lead anywhere ;-)

Reply:
Currently this does not work so i will go with the mailto link, that should work i guess.